src/Ztm.Data.Entity/Contexts/MainDatabase.cs

```diff
@@ -136,12 +136,12 @@ protected virtual void ConfigureWebApiCallbackHistory(ModelBuilder modelBuilder)
         {
             modelBuilder.Entity<WebApiCallbackHistory>(b =>
             {
-                b.Property(e => e.CallbackId).IsRequired().ValueGeneratedNever();
+                b.HasKey(e => e.Id);
+                b.Property(e => e.CallbackId).IsRequired();
                 b.Property(e => e.Status).IsRequired();
                 b.Property(e => e.InvokedTime).IsRequired();
-                b.Property(e => e.Data).HasColumnType("jsonb").IsRequired();
+                b.Property(e => e.Data).IsRequired();
 
-                b.HasKey(e => new { e.CallbackId, e.InvokedTime });
                 b.HasOne(e => e.Callback)
```

Comment:
We need to setup unique index using `CallbackId` here.

Reply:
It should not be unique